s29741-pj-JakubKolad/30.09/Z1.txt

```diff
@@ -0,0 +1,50 @@
+1.Uruchom terminal.ctrl+alt+t
+
+2.Wyświetl zawartość aktualnego katalogu.ls
+
+3.Sprawdź gdzie aktualnie się znajdujesz.pwd
+
+4.Przejdź do katalogu Pulpit za pomocą ścieżki względnej.cd Desktop/
+
+5.Sprawdź gdzie jesteś.pwd
+
+6.Wyświetl zawartość bieżącego katalogu.ls
+
+7.Wyświetl zawartość swojego katalogu domowego bez przechodzenia do niego (ls z odpowiednim parametrem).ls /home
+
+8.Przejdź do katalogu Obrazy (lub innego katalogu, który jest dostępny - podaj wprost, że zmieniasz katalog na inny bo np. Obrazy nie są u Ciebie dostępne) w twoim katalogu domowym za pomocą ścieżki względnej. cd Pictures/
+
+9.Sprawdź gdzie jesteś.pwd
+
+10.Zobacz do jakiego katalogu przejdziesz za pomocą ścieżki ././././././. Katalog nie uległ zmianie.
+
+*	W opisie rozwiązania proszę umieścić również to jak rozumiesz co się właściwie stało.	Dlaczego znalazła/eś się w tym miejscu do którego przeniosła Cię ta ścieżka?
+
+Rozumiem to jako polecenie "zostań tu gdzie jesteś". Po doczytaniu w Internecie rozumiem, że "./" pozwala np. na edytowanie plików "w dole" ścieżki,
+w której się obecnie znajdujemy. Innymi słowy jeżeli plik znajduje się w podkatalogach mojej obecnej lokalizacji to za pomocą "./" nie muszę ręcznie
+do tych katalogów przechodzić żeby go edytować.
+
+*	Czy dodanie kilku ./././././ na początku/środku/końcu powoduje, że ścieżka się zmieni?
+Nie.
+
+*	Co ta ścieżka oznacza/sobą reprezentuje?
+Umożliwia wykonywanie poleceń bez potrzeby zmiany bieżącego katalogu.
+
+11.Przejdź do swojego katalogu domowego za pomocą ścieżki bezwzględnej.
+cd /home
```

Comment:
Do poprawy. To nie jest ścieżka Twojego konkretnego katalogu domowego tylko katalow wszystkich profili użytkowników. Twój jest jednym z nich ale mimo wszystko trzeba go wskazać.